home.html

```diff
@@ -144,15 +144,6 @@ <h5 class="card-title  text-center color-text-brownish"><b>Pizza Funghi</b></h5>
       </div>
     </div>
 
-    <!-- Optional JavaScript; choose one of the two! -->
-  
-    <!-- Option 1: Bootstrap Bundle with Popper -->
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
```

Comment:
I think you need this script for bootstrap to function

Reply:
No this script is not for bootstrap to function but rather certain funcionalities to work. Especially Animations like dropdown menus and stuff, since we dont yet have that we dont need to keep it. If we decide to add smth like that we can add the line later one again

Reply:
It will probably result in someone spending 3 hours trying to understand why the animation isn't playing, or just minor visual problems that won't be noticed.
If you are sure that there are no problems because of it, and you are confident that we won't forget to add it back, then we can remove it.

Reply:
this shouldn’t be deleted the responsive navbar will not work if deleted